packages/openapi-fetch/src/index.ts

```diff
@@ -54,6 +56,31 @@ export type FetchResponse<T> =
   | { data: T extends { responses: any } ? NonNullable<FilterKeys<Success<T["responses"]>, JSONLike>> : unknown; error?: never; response: Response }
   | { data?: never; error: T extends { responses: any } ? NonNullable<FilterKeys<Error<T["responses"]>, JSONLike>> : unknown; response: Response };
 
+/** Call URLSearchParams() on the object, but remove `undefined` and `null` params */
```

Comment:
Minor QoL improvement: yes, the library did use `new URLSearchParams()` as the default. But handling `undefined` / `null` params is annoying to have to implement one’s own `querySerializer` for.